frontend/deploy/helm/frontend/templates/acrpullbinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: acrpull.microsoft.com/v1beta2
+kind: AcrPullBinding
+metadata:
+  name: pull-binding
+spec:
+  acr:
+    environment: PublicCloud
+    scope: {{ .Values.pullBinding.registry }}
+    server: {{ .Values.pullBinding.scope }}
+  auth:
+    workloadIdentity:
+      serviceAccountRef: frontend
+      clientID: {{ .Values.pullBinding.workloadIdentityClientId }}
+      tenantID: {{ .Values.pullBinding.workloadIdentityTenantId }}
```

Comment:
Newest version of the controller allows us to reference the puller's MI for this token, letting multiple MI to be federated to the SA